src/lighteval/metrics/harness_compatibility/drop.py

```diff
@@ -147,10 +160,8 @@ def _normalize(answer):
     max_em = 0
     max_f1 = 0
     for gold_answer in formatted_doc.specific["golds_no_preprocessing"]:
-        if isinstance(gold_answer, list):
-            gold_answer = gold_answer[0]
         exact_match, f1_score = _get_metrics(predictions, gold_answer)
-        if gold_answer.strip():
+        if gold_answer[0].strip():
```

Comment:
if gold_answer is a string, this will fail

Reply:
A simplest solution is to add the check on list after we got the metric results, see suggestion